app/_hub/kong-inc/graphql-rate-limiting-advanced/index.md

```diff
@@ -4,9 +4,10 @@ name: GraphQL Rate Limiting Advanced
 publisher: Kong Inc.
 version: 1.3-x
 
-desc: Provide rate-limiting for GraphQL queries.
+desc: Provide rate limiting for GraphQL queries.
```

Comment:
```suggestion
desc: Provide rate limiting for GraphQL queries
```
This is the short description for the cards on the main hub page, they don't have periods